source/_lovelace/picture-glance.markdown

```diff
@@ -2,128 +2,96 @@
-
-Picture glance supports a display of maximum 10 items.
+Show an image card and corresponding entity states as icon. The entities on the right side allow toggle actions, others show the more-info-dialog.
 
 <p class='img'>
-<img src='/images/lovelace/lovelace_picture_glance.gif' alt='Screenshot of the picture glance card'>
-Screenshot of the picture glance card.
+  <img src='/images/lovelace/lovelace_picture_glance.gif' alt='Picture glance card for a living room'>
+  Picture glance card for a living room.
 </p>
 
 {% configuration %}
 type:
   required: true
   description: picture-glance
   type: string
-image:
+entities:
   required: true
-  description: The URL of an image.
+  description: List of entities.
+  type: list
+title:
+  required: false
+  description: The card title.
   type: string
 navigation_path:
   required: false
-  description: Path of URL to use in navigation.
+  description: Navigate to path on tap action.
   type: string
-entities:
-  required: true
-  description: A list of entity IDs.
-  type: list
-navigation_path:
+image:
   required: false
-  description: Path of URL to use in navigation.
+  description: Background image URL.
   type: string
 camera_image:
   required: false
-  description: The entity ID of a camera.
+  description: Camera entity as Background image.
   type: string
 state_image:
   required: false
-  description: Path of URL to use in navigation.
-  type: list
+  description: Background image based on entity state.
+  type: object
   keys:
-    "on":
-      type: string
-      required: false
-      description: URL of an image used for on state.
-    "off":
+    state:
       type: string
       required: false
-      description: URL of an image used for off state.
-    home:
-      type: string
-      required: false
-      description: URL of an image used for home state.
-    not_home:
-      type: string
-      required: false
-      description: URL of an image used for not_home state.
-    ANYTHING:
-      type: string
-      required: false
-      description: Any state that is supported by the entity works.
+      description: "`state: image-url`, check the example below."
 entity:
   required: false
-  description: "An entity to use for state_image state."
-  type: list
-title:
-  required: false
-  description: The card title.
+  description: Entity to use for `state_image`.
   type: string
 {% endconfiguration %}
 
 ## {% linkable_title Examples %}
 
 ```yaml
 - type: picture-glance
-  image: https://images.pexels.com/photos/276724/pexels-photo-276724.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=240&w=495
-  title: Living
+  title: Living room
   entities:
     - switch.decorative_lights
     - light.ceiling_lights
     - lock.front_door
     - binary_sensor.movement_backyard
     - binary_sensor.basement_floor_wet
+  image: /local/living_room.png
```

Comment:
Images should be uploaded to ‘/images/lovelace/’

Reply:
@sjabby Those are just placeholders. We don't necessarily need those images.